test/e2e/suite/conformance/certificatesigningrequests/tests.go

```diff
@@ -317,35 +359,30 @@ func (s *Suite) Define() {
 				requiredFeatures: []featureset.Feature{featureset.URISANsFeature, featureset.OnlySAN},
 			},
 			{
-				name:    "should issue a certificate that includes arbitrary key usages",
+				name:    "should issue a certificate that includes arbitrary key usages with common name",
 				keyAlgo: x509.RSA,
-				csrModifiers: func() []gen.CSRModifier {
-					return []gen.CSRModifier{
-						gen.SetCSRCommonName(sharedCommonName),
-						gen.SetCSRDNSNames(sharedCommonName),
```

Comment:
There was no reason for `gen.SetCSRDNSNames(` to be set here, it does not have any effect on the usages.